.github/workflows/pr.yaml

```
@@ -35,6 +35,32 @@ jobs:
         with:
           version: nightly
 
+      - name: Use Node.js 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Can we use either an active or the latest NodeJS version instead?
```suggestion
          node-version: latest
```
or, better consider adding a `.nvmrc` file and use the `node-version-file` property instead; something like:
```suggestion
          node-version-file: "./web3-functions/.nvmrc"
```

---

Will be addressed in separate [Issue](https://github.com/LiskHQ/lisk-contracts/issues/210)